libs/klv/src/main/java/org/codice/alliance/libs/klv/FrameCenterKlvProcessor.java

```diff
@@ -67,8 +79,55 @@ private void setAttribute(Metacard metacard, String wkt) {
         metacard.setAttribute(new AttributeImpl(ATTRIBUTE_NAME, wkt));
     }
 
+    private void callFirstHandler(Metacard metacard, List<LatitudeLongitudeHandler> stanagHandlers,
+            Configuration configuration) {
+
+        Integer subsampleCount = (Integer) configuration.get(Configuration.SUBSAMPLE_COUNT);
+
+        stanagHandlers.stream()
+                .findFirst()
+                .ifPresent(handler -> handler.asSubsampledHandler(subsampleCount)
+                        .asAttribute()
+                        .ifPresent(attribute -> doProcess(attribute, metacard)));
+    }
+
+    private boolean areAllHandlersFound(List<LatitudeLongitudeHandler> stanagHandlers) {
+        return stanagHandlers.size() == stanagFieldNames.size();
```

Comment:
Is equivalent size the same as all handlers being found? Are there only the two types of lat/long handlers?

Also, it would save you a stack call to create a const int of `STANAG_FLD_NAME_SIZE`.